### PR TITLE
Task 11

### PR DESCRIPTION
**Pull request by AnnaSovpel**

### CODE REVIEW

File: task_11/task_11.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <div class="main_header">
+        <div class="cell">Name</div>
+        <div class="cell">Owner</div>
+        <div class="actions">Actions</div>
+  </div>
+</header>
+
+<main style="margin-top: 50px;">
+
+<div class="main">
```
Comment (Killerkip10) [inline task_11/task_11.html:20]: What is the "main"?) May be "row" or "table-row"?)

File: task_11/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    top: 0px;
+    height: 50px;
+    line-height: 50px;
+    border-bottom: 1px solid grey;
+}
+
+.body {
+    margin: 0;
+}
+.main {
```
Comment (Killerkip10) [inline task_11/style.css:15]: Why main and main_header classes are contain the same styles?)

File: task_11/style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+.main_header{
+    display: flex;
+    height: 50px;
+    align-items: center;
+    padding: 0px 30px;
+    border-bottom: 1px solid grey;
+}
+.cell {
+    width: 30%
```
Comment (Killerkip10) [inline task_11/style.css:31]: You should have used flex-grow

File: task_11/task_11.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      <div class="edit deleted">Deleted</div>
+    </div>
+</div>
+
+<div class="main" >
+    <div class="cell">Ariel</div>
+    <div class="cell">USA</div>
+    <div class="buttons">
+      <div class="edit">Edit</div>
+      <div class="edit deleted">Deleted</div>
```
Comment (Killerkip10) [inline task_11/task_11.html:52]: "edit" then "delete"
Not understandable

You should have created the common class for button and the modifiers difference.

**Review by Killerkip10 — COMMENTED** [review]
## Task 11
- Html validator errors. (-5 points);
- 1.5 (-20 points);

### Total 75 points

File: task_11/style.css (hunk 2 above)
Comment (AnnaSovpel) [reply task_11/style.css:15]: Done.

**Pull request merged**